File: backend/routes/report_issue_routes.go
package routes

import (
	"eunity.com/backend-main/controllers"
	"github.com/gin-gonic/gin"
)

var Report_Issue_controllers = controllers.Report_Issue_controllers{}

func Report_Issue_routes(r *gin.RouterGroup) {

	//Return Unsubmitted Report
	r.GET("/get_report", Report_Issue_controllers.GET_Get_Issue)

	//Add a New Report
	r.POST("/add_report", Report_Issue_controllers.POST_Add_Issue)

	//Update the existing Unsubmitted Report
	r.PATCH("/update_report", Report_Issue_controllers.PATCH_Update_Issue)

	//Submit the current unsubmitted report
	r.PATCH("/submit_report", Report_Issue_controllers.PATCH_Submit_Report)
}

File: python/routes/report_issue_routes.py
from fastapi import APIRouter, Depends
from controllers import report_issue_controllers as report_issue_controller

router = APIRouter(prefix="/report")



@router.get("/get_report")
async def get_report():
    return report_issue_controller.get_issue()

@router.post("/add_report")
async def add_report():
    return report_issue_controller.add_issue()

@router.patch("/update_report")
async def update_report():
    return report_issue_controller.update_issue()

@router.patch("/submit_report")
async def submit_report():
    return report_issue_controller.submit_report()